### PR TITLE
use etc override config for kurento and pads

### PR DESCRIPTION
**Pull request by amg-web**
use etc override config for kurento and shared notes URL if used or standard config if not.

### CODE REVIEW

File: bigbluebutton-config/bin/bbb-conf [hunk 1]
```diff
@@ -1668,7 +1668,10 @@ if [ -n "$HOST" ]; then
     #
     # Update HTML5 client
     #
-    if [ -f $HTML5_DEFAULT_CONFIG ]; then
+    if [ -f $HTML5_ETC_CONFIG ]; then
+        yq w -i $HTML5_ETC_CONFIG public.kurento.wsUrl "wss://$HOST/bbb-webrtc-sfu"
+        yq w -i $HTML5_ETC_CONFIG public.pads.url      "$PROTOCOL://$HOST/pad"
+    else
```
Comment (schrd) [inline bigbluebutton-config/bin/bbb-conf:1674]: Good catch. I think `$HTML5_DEFAULT_CONFIG` should not be changed at all. Instead `$HTML5_ETC_CONFIG` should be created if it does not exist. Mentioning #14484 as there are probably more things to fix

**Comment by MaximKhlobystov** [comment]
Hi @amg-web,

Thank you for this contribution! Could you please confirm if you already sent in the signed Contributor License Agreement? See https://docs.bigbluebutton.org/support/faq.html#why-do-i-need-to-sign-a-contributor-license-agreement-to-contribute-source-code. Thanks in advance!

**Comment by amg-web** [comment]
@MaximKhlobystov 
I do not have possiblity to sing it in a such way at the moment.

**Comment by amg-web** [comment]
> Hi @amg-web,
> 
> Thank you for this contribution! Could you please confirm if you already sent in the signed Contributor License Agreement? See https://docs.bigbluebutton.org/support/faq.html#why-do-i-need-to-sign-a-contributor-license-agreement-to-contribute-source-code. Thanks in advance!

I will remove PR as I do not have tech possibility to sign agreement

**Comment by ffdixon** [comment]
Confirmed receipt of CLA -- thanks1

**Pull request closed without merging**